lambda_function.py

```diff
@@ -65,5 +65,7 @@ def lambda_handler(event, context):
 
     # Valid event, and one we are interested in
     cluster = event["detail"]["clusterArn"]
-    adjust_service_desired_count(ecs_client(), cluster, service)
+    serviceItems = service.split(',')
+    for item in serviceItems[:]:
```

Comment:
I'm unfamiliar with the syntax of `serviceItems[:]` - can you explain how that is different from `for item in serviceItems:`?